api/src/test/java/uk/gov/hmcts/payment/api/controllers/CaseControllerTest.java

```diff
@@ -261,11 +272,9 @@ public void searchAllPaymentsWithCcdCaseNumberShouldReturnRequiredFieldsForVisua
 
         assertThat(payment.getCcdCaseNumber()).isEqualTo("ccdCaseNumber1");
 
-        assertThat(payment.getReference()).isNotBlank();
+        assertThat(payment.getPaymentReference() == null || !payment.getPaymentReference().trim().isEmpty()).isTrue();
```

Comment:
That looks like a catch all assert - either null or non-empty.. basically anything

Reply:
I could assert for the exact value of paymentReference instead?